### PR TITLE
Show table

### PR DESCRIPTION
**Pull request by Tony-Leng**
Created the table for set_ups show. Currently the "no appointment" status is in the same column as the dates.

image

### CODE REVIEW

File: app/views/set_ups/show.html.erb [hunk 1]
```diff
@@ -1,2 +1,39 @@
...
+<br>
+<table class="mt-4">
+  <tr>
+    <th>Employee Name</th>
+    <th>Health Check Date</th>
+    <th>Status</th>
+  </tr>
+  <% @set_up.company.users.each do |user| %>
+    <tr>
+      <td class="employeename">
```
Comment (tuoanhnt95) [inline app/views/set_ups/show.html.erb]: Is the style (font, color, spacing, etc.) of the employee name different from the style of the health check date? If they are the same, I think we can do one of two things:
opt1: class "employeename" and "healthcheckdate" can use the same class, ex. "table-content-text". 
opt2: class "table-primary-color" with a style and color for the whole table's content text

File: app/views/set_ups/show.html.erb [hunk 2]
```diff
@@ -1,2 +1,39 @@
...
+        <%= "#{user.last_name} #{user.first_name}" %>
+      </td>
+      <% if user.health_checks.find_by(set_up: @set_up) == nil %>
+        <td class="healthcheckdate">
```
Comment (tuoanhnt95) [inline app/views/set_ups/show.html.erb]: Same as above

File: app/views/set_ups/show.html.erb [hunk 3]
```diff
@@ -1,2 +1,39 @@
...
+          <p >No appointment</p>
+        </td>
+      <% else %>
+        <td class="healthcheckdate">
```
Comment (tuoanhnt95) [inline app/views/set_ups/show.html.erb]: Same as above

**Review by tuoanhnt95 — COMMENTED** [review]
Could you check my questions on the table class? If it makes sense, you can remove the classes, and I will add them when I do the css

**Comment by Tony-Leng** [comment]
I removed the table classes. Also, there is a mt-4 on the table, feel free to take it out if you want to style margin as well.
image

**Pull request merged**